migration/migrator/main.py

```diff
@@ -281,6 +281,9 @@ def handle_migration(args):
         if missing_migration.exists():
             missing_migration.unlink()
 
+    if 'master' in args.environments:
```

Comment:
Given that `load_triggers` has its own check on `args.environments`, I think you can get rid of this if statement altogether.